services/graphite/parser_test.go

```diff
@@ -602,6 +602,27 @@ func TestApplyTemplateSpecific(t *testing.T) {
 	}
 }
 
+// Test that most specific template is N/A
```

Comment:
Does this mean that "the most specific template is not applicable? not applied? doesn't matter?" The comment is a little ambiguous.